packages/dashboard-core-plugins/src/panels/FileExplorerPanel.tsx

```diff
@@ -262,6 +262,7 @@ export class FileExplorerPanel extends React.Component<
             onDelete={this.handleDelete}
             onRename={this.handleRename}
             onSelect={this.handleFileSelect}
+            isNewItemModalClicked={false}
```

Comment:
This prop is always `false`, and i don't think is necessary. You should be able to remove it from all classes